common/archiver/s3store/query_parser.go

```diff
@@ -70,18 +68,14 @@ const (
 	PrecisionMinute = "Minute"
 	PrecisionSecond = "Second"
 )
-const (
-	queryTemplate         = "select * from dummy where %s"
-	defaultDateTimeFormat = time.RFC3339
-)
 
 // NewQueryParser creates a new query parser for filestore
 func NewQueryParser() QueryParser {
 	return &queryParser{}
 }
 
 func (p *queryParser) Parse(query string) (*parsedQuery, error) {
-	stmt, err := sqlparser.Parse(fmt.Sprintf(queryTemplate, query))
+	stmt, err := sqlparser.Parse(fmt.Sprintf(util.QueryTemplate, query))
```

Comment:
I would make this a function and unexport `QueryTemplate`:
```suggestion
	stmt, err := sqlquery.Parse(query)
```

Reply:
sqlparser is a 3rd party package. I can't make this change

Reply:
This is not what I ask for :-)